types/nanomsg/index.d.ts

```diff
@@ -1,36 +1,148 @@
-// Type definitions for nanomsg 3.2
+// Type definitions for nanomsg 4.2
 // Project: https://github.com/nickdesaulniers/node-nanomsg
-// Definitions by: Titan <https://github.com/titan>
+// Definitions by: Tobey Blaber <https://github.com/toebeann>
+//                 Titan <https://github.com/titan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
+import { EventEmitter } from "events";
+
 export interface Options {
-  raw?: boolean | undefined;
```

Comment:
These were written like this on purpose—`raw?: boolean` and `raw?: boolean | undefined` mean different things in `--exactOptionalPropertyTypes`.

Reply:
I see, I'm not familiar with that flag. Thought it was just redundant semantics, my bad - will fix up :)

Reply:
Resolved by [bab25b0](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61223/commits/bab25b0d4468e81d61889199b0bdef01786e78a1)